src/planscape/forsys/views.py

```diff
@@ -209,9 +210,10 @@ def run_forsys_generate_project_areas_for_a_single_scenario(
         robjects.StrVector(forsys_condition_headers),
         robjects.FloatVector(forsys_priority_weights),
         forsys_stand_id_header, forsys_proj_id_header, forsys_area_header,
-        forsys_cost_header, forsys_geo_wkt_header,
-        "" if output_scenario_name is None else output_scenario_name,
-        "" if output_scenario_tag is None else output_scenario_tag)
+        forsys_cost_header, forsys_geo_wkt_header, ""
+        if output_scenario_name is None else output_scenario_name, ""
+        if output_scenario_tag is None else output_scenario_tag,
+        enable_kmeans_clustering)
```

Comment:
I see you're using a bunch of other input parameters for `hclust` from the URL now, e.g. cluster size. Are we okay to test kmeans with somewhat hard-coded inputs for now, or should we pass the same parameters to the kmeans flow? Don't want to do premature optimization but also don't want to make testing really complicated. What do you think?

Reply:
I don't have strong opinion on that.

But I do think we will eventually want to pass the number of clusters as a parameter from Planscape to R. This is because we can make runtime estimates as function the number of stands. If we have too many stands, we can cluster to the number of desired stands that'd match our desired runtime.